internal/appsec/config.go

```diff
@@ -109,6 +122,31 @@ func readRateLimitConfig() (rate uint) {
 	return uint(parsed)
 }
 
+func readObfuscatorConfig() ObfuscatorConfig {
+	keyRegex, kPresent := os.LookupEnv(obfuscatorKeyEnvVar)
+	valueRegex, vPresent := os.LookupEnv(obfuscatorValueEnvVar)
+
+	if keyRegex == "" && !kPresent {
+		log.Info("appsec: starting with the default regexp for matched parameter keys obfuscation")
```

Comment:
I think all those Info logs should rather be Debug logs. I try to keep the Info logs to what really matters to the user, so only about things they should see and be aware of by default (since the log is in Info level by default). 

At some point, we could also log the overall appsec configuration struct too like the tracer does.

Reply:
I think this circles back to my original comment. Should we consider this important information?
An application not running the expected regexp to catch malicious attacks sounds like something that may be considered worth informing the user about from the start. Or remove the info logs and break an error if the regexp doesn't compile. WDYT?

Reply:
Yes, exactly what you said at the end: just log an error if a regexp doesn't compile. No need for info logs here for now.

Reply:
Replaced Info logs by Debug and kept the Error logs for compile errors